docs/howto/filesystem-management/filesystem-backups/restore-filesystem.md

```diff
@@ -25,18 +25,24 @@ To restore a home directory from a snapshot, we need to create a new EBS volume
 Please follow AWS's guidance for [restoring EBS volumes from a snapshot](https://docs.aws.amazon.com/prescriptive-guidance/latest/backup-recovery/restore.html#restore-files) to create a new EBS volume from the snapshot.
 ```
 
-Once we have created a new EBS volume from the snapshot, we need to mount it to the EC2 instance attached to the existing EBS volume containing the NFS home directories. To do this, we need to find the instance ID of the EC2 instance attached to the existing EBS volume. This involves the following steps:
+Once we have created a new EBS volume from the snapshot, we can use the `deployer exec root-homes` command to mount the new EBS volume to a pod along with the existing NFS home directories volume.
 
-1. Go to the EBS volumes page in the AWS console
-2. Find the volume ID of the existing EBS volume containing the NFS home directories
-3. Click on the volume ID and find the instance ID in the "Attached resources" section
-4. Once we have the instance ID, we can mount the new EBS volume to the EC2 instance by following the steps outlined in the [Attaching an Amazon EBS volume to an instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-attaching-volume.html) guide.
+```bash
+deployer exec root-homes $CLUSTER_NAME $HUB_NAME --restore-volume-id=<new-ebs-volume-id> --restore-mount-path=/restore-volume --restore-volume-size=100Gi
+```
```

Comment:
```suggestion
deployer exec root-homes $CLUSTER_NAME $HUB_NAME --additional-volume-id=<new-ebs-volume-id> --additional-mount-path=/new-ebs-volume --additional-volume-size=100Gi
```
```

Reply:
@sgibson91 updated the docs to match the new CLI options.